gui/wxpython/startup/guiutils.py

```diff
@@ -324,13 +331,19 @@ def download_location_interactively(guiparent, grassdb):
     """
     from startup.locdownload import LocationDownloadDialog
 
+    loc_download_output = (None, None, "PERNAMENT")
```

Comment:
Just naming, the variable name is unnecessarily descriptive. You are in a relatively short `download_location...` function. Simply calling it, e.g., `result` in the sense "this is what will be used as a return value" is enough and likely best describes the variable in its context which is this function.

Reply:
Changed to **result**.